iothub/service/samples/getting started/FileUploadNotificationReceiverSample/Parameters.cs

```diff
@@ -8,6 +8,13 @@ namespace Microsoft.Azure.Devices.Samples
     /// <summary>
     /// Parameters for the application.
     /// </summary>
+
+    public enum Transport
+    {
+        Amqp_WebSocket_Only = IotHubTransportProtocol.WebSocket,
+        Amqp = IotHubTransportProtocol.Tcp
+    };
```

Comment:
No reason to create this class. Just have a parameter ofr IotHubTransportProtocol.